src/terarangerone.cpp

```diff
@@ -99,13 +117,6 @@ void TerarangerOne::serialDataCallback(uint8_t single_character)
   static int buffer_ctr = 0;
   static int seq_ctr = 0;
 
-  sensor_msgs::Range range_msg;
-  range_msg.field_of_view = 0.0593;
-  range_msg.max_range = 14.0;
-  range_msg.min_range = 0.2;
-  range_msg.header.frame_id = "base_range";
-  range_msg.radiation_type = sensor_msgs::Range::INFRARED;
-
```

Comment:
instantiate range message and populate static fields only once in class constructor